src/components/task-edit.js

```diff
@@ -1,6 +1,109 @@
-export const createCardTaskEditTemplate = () => {
+import {COLORS, DAYS, MONTH_NAMES} from '../const.js';
+import {formatTime} from '../utils.js';
+
+// функция создания хештега
+const createHashtagMarkup = (tag) => {
   return (
-    `<article class="card card--edit card--yellow card--repeat">
+    `<span class="card__hashtag-inner">
+      <input
+        type="hidden"
+        name="hashtag"
+        value=${tag}
+        class="card__hashtag-hidden-input"
+      />
+      <p class="card__hashtag-name">
+        #${tag}
+      </p>
+      <button
+          type="button"
+          class="card__hashtag-delete"
+      >
+        delete
+      </button>
+    </span>`
+  );
+};
+
+// функция вывода хештегов
+const createHashtags = (tags) => {
+  return Array.from(tags)
+    .map((tag) => createHashtagMarkup(tag))
```

Comment:
```suggestion
    .map(createHashtagMarkup)
```
Отлично! Можно даже проще

Reply:
Круто, буду иметь в виду на будущее.